src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/idam/cache/IdamCachedClientTest.java

```diff
@@ -196,4 +206,56 @@ void should_create_new_token_when_token_removed_from_cache() {
         verify(users, times(2)).getUser(any());
         verify(idamClient, times(2)).authenticateUser(any(), any());
     }
+
+
+    @Test
+    public void should_get_userDetail_when_no_error() {
+        given(idamClient.getUserDetails(JWT)).willReturn(USER_DETAILS);
+
+        UserDetails userDetails1 =
+            idamCachedClient.getUserDetails(JWT);
+
+        assertThat(userDetails1).usingRecursiveComparison().isEqualTo(USER_DETAILS);
+        verify(idamClient).getUserDetails(any());
```

Comment:
Just in case, best to verify agains the expected user name.

Reply:
I just want to very getUserDetails method called only once with any value